app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,24 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
- [ ] Even after running the `./bin/dev` command you recommended, the problem the previous reviewer highlighted still persists. As you can see, the error is a result of tailwind.css not being present in the asset pipeline. Please see the error below and try to fix it 🙏 
![image](https://user-images.githubusercontent.com/46839548/197294842-7f8e1028-9c36-40a5-93d1-93e3443b4d3f.png)

Reply:
I cloned the project again, and ran the `./bin/dev` command and it fixed it.